README.mkd

```diff
@@ -1,54 +1,49 @@
 This is a package for PhotoColleSDK for iOS.
 PhotoColleSDK is a SDK to use docomo PHOTO COLLECTION services.
 
-This package contains followings:
+[![CocoaPods](https://img.shields.io/badge/CocoaPods-released-green.svg)](http://cocoapods.org/pods/PhotoColleSDK)
 
-  * Reference guide and API references in docs directory.
-  * PhotoColleSDK source files in photocolle-sdk directory.
-  * Test application source files in test-app directory.
-  * XIB file for PhotoColleSDK.
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-green.svg)](https://github.com/Carthage/Carthage)
 
-Please read README.mkd under each directory for details.
 
-## photocolle\_sdk\_setting.plist
+## Documentations
 
-You can change access URL by creating photocolle\_sdk\_setting.plist
-on your application's resource directory.
+### API Documentation
 
-You can get debug log for PhotoColleSDK by adding ```logging```
-property into photocolle\_sdk\_setting.plist.
+Please see the [API
+documentation](http://cocoadocs.org/docsets/PhotoColleSDK/1.1.1/) for
+more details
 
-Contents of photocolle\_sdk\_setting.plist are followings:
+### Reference Guide
 
-  * "authorityUrl" is base URL for authentication server.
-  * "tokenUrl" is base URL for a server to get access token.
-  * "photocolleUrl" is base URL for PhotoColle server.
-  * "logging" enables debug log, if logging is true, PhotoColleSDK output debug log. Otherwise PhotoColleSDK does not output debug log. Debug logs are outputed by NSLog method.
+Please see the [reference guide](docs/REFERENCE_GUIDE.mkd)
 
-example:
+## Installation
 
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>authorityUrl</key>
-	<string>https://authority.example.com</string>
-	<key>tokenUrl</key>
-	<string>https://token.example.com</string>
-	<key>photocolleUrl</key>
-	<string>https://service.example.com</string>
-	<key>logging</key>
-	<true/>
-</dict>
-</plist>
+### CocoaPods
+
+PhotoColleSDK is placed on [CocoaPods](http://cocoapods.org/?q=photocollesdk).
+Please add a following in your Podfile:
+
+```
+pod 'PhotoColleSDK', '>= 1.1.1'
+```
+
+### Carthage
+
+PhotoColleSDK is compatible with __Carthage__. You can learn how to
+install with __Carthage__ by [Getting Started Guide for
+iOS](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos).
+
+```
+github "KiiPlatform/photocolle-iOSSDK" >= 1.1.1
 ```
 
-## Add XIB file to your application project.
+### Build this framework
 
-To use PhotoColleSDK, You need to add DCOAuth2View.xib file to your
-application project. DCOAuth2View.xib is placed on top directory of
-this distribution package.
+You can build PhotoColleSDK.framework by your
+own. [README.mkd](photocolle-sdk/README.mkd) tells you how to build
+this framework.
```

Comment:
@yongpingchen 

I think that it is better to separate these 2 READMEs.

Almost all of application developers must not be interested in how to build framework. Many developers must use CocoaPods or Carthage. Describing how to build is boring for these developers I think.

But this description is not good. `README.mkd` may confuse readers of this documentation.

I fixed at 3a9904a. What do you think this fix?